app/views/cars/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "My cars") %>
+<ul class="list-unstyled">
+  <button class="button">
+    <%= link_to "Add new car", new_car_path %>
```

Comment:
What about internationalization? We shouldn't keep hard-coded strings in template files directly

Reply:
I guess, on this phase of development we can not focus on internationalization. But it's definitely good idea to extract this string to variable to ease later refactorization.